admin/app.js

```diff
@@ -19,9 +19,23 @@ const session = require('express-session')
 const MongoStore = require('connect-mongo')(session)
 const breadcrumbs = require('express-breadcrumbs')
 const fs = require('fs')
+const config = require('./config')
+const whitelist = require('./whitelist')
+
+const unsetVars = []
+Object.keys(config).map((key) => {
+  if (!config[key] && !whitelist.includes(key)) {
+    unsetVars.push(`${key}`)
+  }
+})
+
+if (unsetVars.length > 0) {
+  console.log(`The following environment variables need to be defined:\n${unsetVars.join('\n')}`)
+  process.exit()
```

Comment:
According to https://nodejs.org/api/process.html#process_process_exit_code this is misuse.  They reccommend: 

```
if (unsetVars.length > 0) {
  console.log(`The following environment variables need to be defined:\n${unsetVars.join('\n')}`)
  process.exitCode = 1
  throw new Error('Bad config')
}
```

You can get the error from the shell immediately after the process has exited by looking at `$?` E.g. 

`echo $?`

To confirm that the exit code is non-zero (to indicate an error occurred)